CoEpi/AppDelegate.swift

```diff
@@ -11,7 +11,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     internal var window: UIWindow?
     
     private var rootWireframe: RootWireFrame?
-
+    
+    @available(iOS 13.0, *)
```

Comment:
ditto a question on making it iOS12 compatible - it looks like this was needed for ScreenDelegate, which we're not calling - so I should pull this back out? @i-schuetz

Reply:
I imagine it should be fine? As long as you test on iOS 12 and everything still works

Reply:
k, thanks. All of the examples of making iOS12 compatible referenced it without it being internal so wasn't sure if our current use would snafu on iOS12.